docs/articles/documentation/recipes/finding-code-issues-with-flow-type-checker.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+title: Finding Code Issues with Flow Type Checker
+permalink: /documentation/recipes/finding-code-issues-with-flow-type-checker.html
+---
+# Finding Code Issues with Flow Type Checker
+
+[Flow](https://flow.org/en/) is a static type checker for JavaScript. It can help you identify problems in your code
+by infering variable types or analyzing [static type annotations](https://flow.org/en/docs/types/). For more information, see the [Flow documentation](https://flow.org/en/docs/).
+
+This recipe describes how to check TestCafe test code with Flow.
+
+## Step 1 - Install Flow
+
+To use Flow with TestCafe, you will need two packages.
+
+The first one is the Flow binary responsible for checking your code. Install it locally to the project.
+
+```sh
+npm install --save-dev flow-bin
+```
+
+With this module, you can introduce type annotations in your code (or let Flow infer the types implicitly) and run code checks.
+
+However, this only works for your test code. Flow will still treat TestCafe API as untyped and imply that all API members has the type `any`.
+
+To provide type information for third-party libraries (including TestCafe), Flow uses a special module - [flow-typed](https://github.com/flowtype/flow-typed). Install it globally.
+
+```sh
+npm install -g flow-typed
+```
+
+## Step 2 - Initialize Flow
+
+Run the following command to initialize Flow with default configuration.
+
+```sh
+./node_modules/.bin/flow init
```

Comment:
It would be better to configure `npm run flow` than to write `./node_modules/.bin/flow` every time.